examples/coco_example.py

```diff
@@ -0,0 +1,446 @@
+                 g.get_node_by_name(name_of_input_node).output(0))
+  g.remove_node_by_name(name_of_input_node)
+  g.rename_node("raw_image", name_of_input_node)
+
+  # Tack on the postprocessing graph at the original output and rename
+  # the postprocessed output to the original output's name
+  # The original graph produces an output called "detection_classes".
+  # The postprocessing graph goes from "detection_classes" to
+  # "decoded_detection_classes".
+  # The graph after modification produces decoded classes under the original
+  # "detection_classes" name. The original output is renamed to
+  # "raw_detection_classes".
+  g.rename_node("detection_classes", "raw_detection_classes")
+  gde.copy(post_g, g)
+  gde.reroute_ts(g.get_node_by_name("raw_detection_classes").output(0),
+                 g.get_node_by_name("detection_classes").output(0))
+  g.remove_node_by_name("detection_classes")
+  g.rename_node("decoded_detection_classes", "detection_classes")
+
+
+def _apply_graph_transform_tool_rewrites(g: gde.Graph,
+                                         input_node_names: List[str],
+                                         output_node_names: List[str]) \
+        -> tf.GraphDef:
+  """
+  Use the [Graph Transform Tool](
+  https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/
+  graph_transforms/README.md)
+  to perform a series of pre-deployment rewrites.
+
+  Args:
+     g: GDE representation of the core graph.
+     input_node_names: Names of placeholder nodes that are used as inputs to
+       the graph for inference. Placeholders NOT on this list will be
+       considered dead code.
+     output_node_names: Names of nodes that produce tensors that are outputs
+       of the graph for inference purposes. Nodes not necessary to produce
+       these tensors will be considered dead code.
+
+  Returns: GraphDef representation of rewritten graph.
+  """
+  # Invoke the Graph Transform Tool using the undocumented Python APIs under
+  # tensorflow.tools.graph_transforms
+  after_tf_rewrites_graph_def = graph_transforms.TransformGraph(
+    g.to_graph_def(),
+    inputs=input_node_names,
+    outputs=output_node_names,
+    # Use the set of transforms recommended in the README under "Optimizing
+    # for Deployment"
+    transforms=['strip_unused_nodes(type=float, shape="1,299,299,3")',
+                'remove_nodes(op=Identity, op=CheckNumerics)',
+                'fold_constants(ignore_errors=true)',
+                'fold_batch_norms',
+                'fold_old_batch_norms']
+  )
+  return after_tf_rewrites_graph_def
+
+
+def _graph_has_op(g: tf.Graph, op_name: str):
+  """
+  A method that really ought to be part of `tf.Graph`. Returns true of the
+  indicated graph has an op by the indicated name.
+  """
+  all_ops_in_graph = g.get_operations()
+  names_of_all_ops_in_graph = [o.name for o in all_ops_in_graph]
+  return op_name in names_of_all_ops_in_graph
```

Comment:
This could be simplified I think
`return any(op_name == o.name for o in all_ops_in_graph)`

Reply:
Good point, updated.